src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/GithubAuthenticationTests.kt

```diff
@@ -125,26 +126,22 @@ class GithubAuthenticationTests : CustomConfigTests()
 
     private fun assertAuthSuccess(result: Response)
     {
-        result.use {
```

Comment:
this change is because I've changed this `Response` type so it's no longer closeable (instead the response from OkHttp gets closed when we create this: https://github.com/vimc/orderly-web/pull/508/files#diff-cd7bfd9025f911f7b6bc62e1104e07798a28f5ac57a55618b844b38da4fbce13L106)